framework/PostProcessors/SampleSelector.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on August 28, 2018
+
+@author: giovannimaronati
+"""
+from __future__ import division, print_function , unicode_literals, absolute_import
+import warnings
+warnings.simplefilter('default', DeprecationWarning)
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from .PostProcessor import PostProcessor
+from utils import utils
+from utils import InputData
+import Runners
+#Internal Modules End-----------------------------------------------------------
+
+class SampleSelector(PostProcessor):
+  """
+    Constructs a load duration curve.
+    x-axis is time spent above a particular variable's value,
+    y-axis is the value of the variable.
```

Comment:
Docstring copied and pasted from the ValueDuration PP... please add a specific documentation for this class